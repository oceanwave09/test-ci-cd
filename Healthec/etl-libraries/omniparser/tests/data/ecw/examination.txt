apuid|RecordID|PatientId|PatientAccountNo|encounterId|EncounterDate|ProviderNPI|Ecwstructid|Question|QuestionID|Answers|AnswerID|ItemName|ItemID|CategoryName|CategoryID|MainCategoryName|MainCategoryID
280807|280807.17502.2510439.1321|17502|17502|2510439|2022-10-14|1073244257||Start Time:|1321|||Start/End Times|21471|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1321|17502|17502|2510439|2022-10-14|1073244257||Start Time:|1321|||Precautions/Contraindications|21473|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1321|17502|17502|2510439|2022-10-14|1073244257||Start Time:|1321|||Subjective|21474|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1321|17502|17502|2510439|2022-10-14|1073244257||Start Time:|1321|||Speech Therapy Treatment Modalities|21475|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1321|17502|17502|2510439|2022-10-14|1073244257||Start Time:|1321|||NOTE|21476|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1322|17502|17502|2510439|2022-10-14|1073244257||End Time:|1322|||Start/End Times|21471|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1322|17502|17502|2510439|2022-10-14|1073244257||End Time:|1322|||Precautions/Contraindications|21473|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1322|17502|17502|2510439|2022-10-14|1073244257||End Time:|1322|||Subjective|21474|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1322|17502|17502|2510439|2022-10-14|1073244257||End Time:|1322|||Speech Therapy Treatment Modalities|21475|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1322|17502|17502|2510439|2022-10-14|1073244257||End Time:|1322|||NOTE|21476|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1323|17502|17502|2510439|2022-10-14|1073244257||Treatment Modalities:|1323|Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing|2287,2296,|Start/End Times|21471|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1323|17502|17502|2510439|2022-10-14|1073244257||Treatment Modalities:|1323|Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing|2287,2296,|Precautions/Contraindications|21473|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1323|17502|17502|2510439|2022-10-14|1073244257||Treatment Modalities:|1323|Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing|2287,2296,|Subjective|21474|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1323|17502|17502|2510439|2022-10-14|1073244257||Treatment Modalities:|1323|Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing|2287,2296,|Speech Therapy Treatment Modalities|21475|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2510439.1323|17502|17502|2510439|2022-10-14|1073244257||Treatment Modalities:|1323|Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing|2287,2296,|NOTE|21476|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1321|17502|17502|2509800|2022-10-14|1073244257||Start Time:|1321||0|Start/End Times|21471|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1321|17502|17502|2509800|2022-10-14|1073244257||Start Time:|1321||0|Precautions/Contraindications|21473|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1321|17502|17502|2509800|2022-10-14|1073244257||Start Time:|1321||0|Subjective|21474|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1321|17502|17502|2509800|2022-10-14|1073244257||Start Time:|1321||0|Speech Therapy Treatment Modalities|21475|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1321|17502|17502|2509800|2022-10-14|1073244257||Start Time:|1321||0|NOTE|21476|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1322|17502|17502|2509800|2022-10-14|1073244257||End Time:|1322||0|Start/End Times|21471|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1322|17502|17502|2509800|2022-10-14|1073244257||End Time:|1322||0|Precautions/Contraindications|21473|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1322|17502|17502|2509800|2022-10-14|1073244257||End Time:|1322||0|Subjective|21474|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1322|17502|17502|2509800|2022-10-14|1073244257||End Time:|1322||0|Speech Therapy Treatment Modalities|21475|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1322|17502|17502|2509800|2022-10-14|1073244257||End Time:|1322||0|NOTE|21476|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1323|17502|17502|2509800|2022-10-14|1073244257||Treatment Modalities:|1323|Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training|2282,2284,|Start/End Times|21471|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1323|17502|17502|2509800|2022-10-14|1073244257||Treatment Modalities:|1323|Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training|2282,2284,|Precautions/Contraindications|21473|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1323|17502|17502|2509800|2022-10-14|1073244257||Treatment Modalities:|1323|Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training|2282,2284,|Subjective|21474|DAILY NOTE|419371|SPEECH THERAPY|419363
280807|280807.17502.2509800.1323|17502|17502|2509800|2022-10-14|1073244257||Treatment Modalities:|1323|Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training|2282,2284,|Speech Therapy Treatment Modalities|21475|DAILY NOTE|419371|SPEECH THERAPY|419363
