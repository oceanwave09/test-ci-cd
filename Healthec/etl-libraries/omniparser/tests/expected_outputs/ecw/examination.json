[
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21471",
		"item_name": "Start/End Times",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2510439.1321",
		"row_id": "e3d301ed-497b-4989-b51f-28ccc50bde47"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21473",
		"item_name": "Precautions/Contraindications",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2510439.1321",
		"row_id": "d9fc322e-0664-49a7-8801-a738dd7de90b"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21474",
		"item_name": "Subjective",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2510439.1321",
		"row_id": "47fde292-b72b-480b-885a-68bb65750449"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21475",
		"item_name": "Speech Therapy Treatment Modalities",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2510439.1321",
		"row_id": "8f807638-1769-4440-835c-98a40f3730c3"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21476",
		"item_name": "NOTE",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2510439.1321",
		"row_id": "8276ce61-be92-4850-aede-c2989a1fa4a5"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21471",
		"item_name": "Start/End Times",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2510439.1322",
		"row_id": "f6449ecc-b5e4-4208-9e30-ce3f47ca1747"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21473",
		"item_name": "Precautions/Contraindications",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2510439.1322",
		"row_id": "fe4e608f-d568-4f66-a4fc-64cb0e716ffc"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21474",
		"item_name": "Subjective",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2510439.1322",
		"row_id": "f31b1593-0b99-42bc-b39c-be35789db2be"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21475",
		"item_name": "Speech Therapy Treatment Modalities",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2510439.1322",
		"row_id": "155c3327-ea52-4f66-8e63-cdf918eebba7"
	},
	{
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21476",
		"item_name": "NOTE",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2510439.1322",
		"row_id": "12317ea8-de4b-4877-a5af-8ae4800b8836"
	},
	{
		"answers": "Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing",
		"answers_id": "2287,2296,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21471",
		"item_name": "Start/End Times",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2510439.1323",
		"row_id": "fe4b1293-dc87-44a4-9b9e-37f86be60f8e"
	},
	{
		"answers": "Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing",
		"answers_id": "2287,2296,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21473",
		"item_name": "Precautions/Contraindications",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2510439.1323",
		"row_id": "7377946b-661f-45c6-a693-0b2d9b54d1b2"
	},
	{
		"answers": "Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing",
		"answers_id": "2287,2296,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21474",
		"item_name": "Subjective",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2510439.1323",
		"row_id": "90ba0004-cd7d-4bd0-a416-977968609ffc"
	},
	{
		"answers": "Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing",
		"answers_id": "2287,2296,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21475",
		"item_name": "Speech Therapy Treatment Modalities",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2510439.1323",
		"row_id": "8f549ef9-ba56-4a07-a2db-267e5812a8c4"
	},
	{
		"answers": "Articulatory Drills, Conversational Topic Maintenance, Comprehension of WH- Questions, Sequencing",
		"answers_id": "2287,2296,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2510439",
		"item_id": "21476",
		"item_name": "NOTE",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2510439.1323",
		"row_id": "9079ca41-d4cd-4392-891f-cec0be33ec37"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21471",
		"item_name": "Start/End Times",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2509800.1321",
		"row_id": "2d258d52-5d50-4a63-aacc-c748ed30d41a"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21473",
		"item_name": "Precautions/Contraindications",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2509800.1321",
		"row_id": "e57e8e38-f55e-4e42-88c7-cbc9c413c35d"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21474",
		"item_name": "Subjective",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2509800.1321",
		"row_id": "fe096a29-654a-4135-917a-eb3e011a9d8b"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21475",
		"item_name": "Speech Therapy Treatment Modalities",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2509800.1321",
		"row_id": "fff56d4e-2b09-43b4-a558-e51895fe3160"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21476",
		"item_name": "NOTE",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Start Time:",
		"question_id": "1321",
		"record_id": "280807.17502.2509800.1321",
		"row_id": "4883a35b-2945-4638-9d1b-e741f5781ae4"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21471",
		"item_name": "Start/End Times",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2509800.1322",
		"row_id": "f6934e76-eb91-4e26-9154-ff8543b2ab72"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21473",
		"item_name": "Precautions/Contraindications",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2509800.1322",
		"row_id": "ddea46d5-ddee-47ad-b1d7-9b6b98da6947"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21474",
		"item_name": "Subjective",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2509800.1322",
		"row_id": "954fcb21-8bf2-433d-a741-a7791440a04a"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21475",
		"item_name": "Speech Therapy Treatment Modalities",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2509800.1322",
		"row_id": "863273b3-39a6-4250-a8d8-548bc4b6e150"
	},
	{
		"answers_id": "0",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21476",
		"item_name": "NOTE",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "End Time:",
		"question_id": "1322",
		"record_id": "280807.17502.2509800.1322",
		"row_id": "fac1743f-10b5-4770-9171-4be33be6d938"
	},
	{
		"answers": "Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training",
		"answers_id": "2282,2284,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21471",
		"item_name": "Start/End Times",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2509800.1323",
		"row_id": "dedbda15-376f-4ecd-ab41-369049fc3f21"
	},
	{
		"answers": "Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training",
		"answers_id": "2282,2284,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21473",
		"item_name": "Precautions/Contraindications",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2509800.1323",
		"row_id": "681af359-4179-4c1b-b4ff-09818327b3ef"
	},
	{
		"answers": "Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training",
		"answers_id": "2282,2284,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21474",
		"item_name": "Subjective",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2509800.1323",
		"row_id": "17615f9e-15b4-4c85-8cab-a5b068753cea"
	},
	{
		"answers": "Communication Board, Speech Output Device, Vocabulary Expansion, Social Reciprocity/Turn Taking, Joint Attention, Categorization, Comprehension of WH- Questions, Comprehension of Yes/No Questions, Caregiver Training",
		"answers_id": "2282,2284,",
		"apuid": "280807",
		"category_id": "419371",
		"category_name": "DAILY NOTE",
		"encounter_date": "2022-10-14",
		"encounter_id": "2509800",
		"item_id": "21475",
		"item_name": "Speech Therapy Treatment Modalities",
		"main_category_id": "419371",
		"main_category_name": "419371",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "777f496fbe0a7879392e1f288e8bb400",
		"question": "Treatment Modalities:",
		"question_id": "1323",
		"record_id": "280807.17502.2509800.1323",
		"row_id": "39e7e735-d098-4264-a1df-f6fdd6f6cdc2"
	}
]
